src/main/kotlin/it/pagopa/wallet/services/WalletService.kt

```diff
@@ -125,6 +133,156 @@ class WalletService(
+                                                            .type("CARD")
+                                                            .iframeUrl(state.url)
+                                                    )
+                                            }
+                                            .map { response -> response to cardData }
+                                    }
+                                    .map { (response, data) ->
+                                        response to
+                                            Wallet(
+                                                wallet.id,
+                                                wallet.userId,
+                                                WalletStatusDto.VERIFIED,
+                                                wallet.creationDate,
+                                                wallet
+                                                    .updateDate, // TODO update with auto increment
+                                                // with
+                                                // CHK-2028
+                                                wallet.paymentMethodId,
+                                                wallet.paymentInstrumentId,
+                                                wallet.applications,
+                                                wallet.contractId,
+                                                details =
+                                                    DomainCardDetails(
+                                                        Bin(data.bin.orEmpty()),
+                                                        MaskedPan(
+                                                            data.bin.orEmpty() +
+                                                                ("*".repeat(
+                                                                    16 -
+                                                                        data.bin.orEmpty().length -
+                                                                        data.lastFourDigits
+                                                                            .orEmpty()
+                                                                            .length
+                                                                )) +
+                                                                data.lastFourDigits.orEmpty()
+                                                        ),
+                                                        ExpiryDate(data.expiringDate.orEmpty()),
+                                                        WalletCardDetailsDto.BrandEnum.valueOf(
+                                                            data.circuit.orEmpty()
+                                                        ),
+                                                        CardHolderName("?")
```

Comment:
@infantesimone we don't have this info from getCardData. I put here this placeHolder to remember to take a decision about this